macros.in

```diff
@@ -606,19 +611,31 @@ Supplements:   (%{name} = %{version}-%{release} and langpacks-%{1})\
 %_db_backend	      @DB_BACKEND@
 
 #==============================================================================
-# ---- GPG/PGP/PGP5 signature macros.
-#	Macro(s) to hold the arguments passed to GPG/PGP for package
+# ---- OpenPGP signature macros.
+#	Macro(s) to hold the arguments passed to the implementating cmd package
```

Comment:
not native english, but the `implementating` sounds awkward to me here. Should this be `implementation` or something else?
```suggestion
#	Macro(s) to hold the arguments passed to the implementation of cmd package
```

Reply:
Oh, indeed. That word isn't even English and the whole sentence is just gibberish. What it intended to say is "..arguments passed to the cmd implementing package signing" which probably still is a bit awkward but hopefully understandable :sweat_smile: